atom/browser/atom_download_manager_delegate.cc

```diff
@@ -92,7 +384,7 @@ void AtomDownloadManagerDelegate::OnDownloadPathGenerated(
   file_dialog::DialogSettings settings;
   settings.parent_window = window;
   settings.title = item->GetURL().spec();
-  settings.default_path = target_path;
+  settings.default_path = path;
   if (path.empty() && file_dialog::ShowSaveDialog(settings, &path)) {
```

Comment:
@bridiver we should merged https://github.com/brave/muon/pull/385 first and then rebase this PR on it